src/cascadia/TerminalSettingsModel/ActionAndArgs.cpp

```diff
@@ -52,6 +52,7 @@ static constexpr std::string_view SwitchToTabKey{ "switchToTab" };
 static constexpr std::string_view TabSearchKey{ "tabSearch" };
 static constexpr std::string_view ToggleAlwaysOnTopKey{ "toggleAlwaysOnTop" };
 static constexpr std::string_view ToggleCommandPaletteKey{ "commandPalette" };
+static constexpr std::string_view SuggestionsKey{ "suggestions" };
```

Comment:
```suggestion
static constexpr std::string_view SuggestionsKey{ "showSuggestions" };
```
I believe there was discussion in the spec about renaming this. Still like the idea.